nexus/src/saga_interface.rs

```diff
@@ -29,31 +29,32 @@ impl fmt::Debug for SagaContext {
 }
 
 impl SagaContext {
-    pub fn new(
+    pub(crate) fn new(
         nexus: Arc<Nexus>,
         log: Logger,
         authz: Arc<authz::Authz>,
     ) -> SagaContext {
         SagaContext { authz, nexus, log }
     }
 
-    pub fn log(&self) -> &Logger {
+    pub(crate) fn log(&self) -> &Logger {
         &self.log
     }
 
-    pub fn authz(&self) -> &Arc<authz::Authz> {
+    pub(crate) fn authz(&self) -> &Arc<authz::Authz> {
         &self.authz
     }
 
-    pub fn nexus(&self) -> &Arc<Nexus> {
+    pub(crate) fn nexus(&self) -> &Arc<Nexus> {
         &self.nexus
     }
 
-    pub fn datastore(&self) -> &db::DataStore {
+    pub(crate) fn datastore(&self) -> &db::DataStore {
         self.nexus.datastore()
     }
 
-    pub async fn sled_client(
+    #[allow(dead_code)]
```

Comment:
Similar as above - if this is truly dead, could we remove it?

Reply:
Ah, this one truly is dead. Removed.